readme.adoc

```diff
@@ -29,38 +29,36 @@ Table of contents overview:
 Your feedback and contributions are most welcome! Instructions for
 building and contributing can be found in <<appendixbuild>>.
 
+=== What to expect?
+
+Bitcoin is a huge subject and we couldn't possibly cover all of its aspects here. Nevertheless, we
+hope to discuss the necessary features to get you started as well as to enable you to further explore it on your own.
+
+There are lots of people involved in Bitcoin; as some of them have opposing opinions,
+there are resources that express contradictory ideas. However, we always
+attempt to stick to the facts, where opinions do not matter.
+
+=== Who wrote this?
+
+The main authors of this book are Kalle Rosenbaum and Linnéa Rosenbaum. This work
+was commissioned and funded by Chaincode Labs, a development center that runs educational
+programs for developers who want to https://learning.chaincode.com/[learn] about Bitcoin development.
 
 === How is this organized?
 
-The project is sectioned into chapters of different topics. Each
+The project is sectioned into chapters covering different topics. Each
```

Comment:
Maybe we should say "book" instead of "project", now that we're making a book out of it...?

Reply:
Makes sense